multik-api/src/commonMain/kotlin/org/jetbrains/kotlinx/multik/ndarray/complex/_ComplexArrays.kt

```diff
@@ -1501,7 +1504,8 @@ public operator fun ComplexFloatArray.plus(elements: ComplexFloatArray): Complex
     val thisSize = size
     val arraySize = elements.size
     val result = this.copyOf(thisSize + arraySize)
-    System.arraycopy(elements.getFlatArray(), 0, result.getFlatArray(), thisSize * 2, arraySize * 2)
+    val endIndex = thisSize * 2 + arraySize * 2
+    elements.getFlatArray().copyInto(result.getFlatArray(), thisSize * 2,0, endIndex)
```

Comment:
Probably you can reuse `copyInto` on complex arrays, this will eliminate the need to multiply indices by two and call `flatArray`